### DISCUSSION TITLE
关于解密游戏的部分

### DISCUSSION THREAD

**Discussion by AliceCode987** [Meloong-Git/PCL#4165] [提问] [answered]
开原版不是也可以下载MC, 离线启动MC吗? 解密游戏的的代码又是干什么用的? 很好奇为什么要扣掉, 如果是套壳PCL2倒卖的话, 用不到所谓的解密游戏的代码吧?

**Comment by MoYuan-CN** [comment, marked as answer]
## `PCL 正式版/开源版` 和 `PCL 快照版` 的最主要区别：

解锁码系统（主题系统）

---

由于解谜游戏涉及到解锁主题，所以那部分代码被抠掉了（~~绝对不会是因为不想让你们研究的~~）
另外开源版的更多页面由于掺杂了太多相关代码，也被抠掉了 Orz……

**Comment by AliceCode987** [comment]
😂,那你们应该改下README吧,容易产生误解, 那只是PCL本身又不是游戏本体的解密, 主要是我想参考一下PCL2的代码, 还以为又是微软设计的奇妙加密算法不公开, 如果是主题的话倒是无所谓, 谢谢

**Reply by MoYuan-CN** [reply]
确实 README 貌似该修改修改了……
